scipy/interpolate/rbf.py

```diff
@@ -77,15 +89,25 @@ class Rbf(object):
             'cubic': r**3
             'quintic': r**5
             'thin_plate': r**2 * log(r)
+            'bessel': jn(self.p/2-1, 1.0/self.epsilon*r)/(arg**(self.p/2-1))
+            'polyharmonic': r**(2*self.m + 1)
 
         If callable, then it must take 2 arguments (self, r).  The epsilon
         parameter will be available as self.epsilon.  Other keyword
         arguments passed in will be available as well.
-
+    degree : integer, optional
+        Adjustable constant denoting the polynomial degree to include for
+        conditionally positive definite radial functions.
+        - defaults to None, which performs the default RBF interpolation
+        without any additional generated coefficients.
     epsilon : float, optional
         Adjustable constant for gaussian or multiquadrics functions
         - defaults to approximate average distance between nodes (which is
         a good start).
+    m : float, optional
+        Adjustable constant for polyharmonic splines.
```

Comment:
Again, what is m, how do you adjust it?  Guidelines here would be helpful to the user.